Documentation/coding-guidelines/buildingvertical.md

```diff
@@ -2,30 +2,29 @@
 
 **Definitions**
 
-*VerticalGroup*
+*BuildConfiguration*
 
-`VerticalGroup` - We need a property to define the vertical target group, but we don't want to set "TargetGroup" explicitly or we won't be able to build the "" TargetGroups for projects.
-  If `VerticalGroup != ""`, we import buildvertical.targets which will contain our additional targets.
+`BuildConfiguration` - We need a property to define the configuration we plan to build the entire repo with.
 
-*ProjectConfigurations*
+*BuildConfigurations*
 
-For each ref project and src project, we define `ProjectConfigurations`. `ProjectConfigurations` is a tuple for the supported `TargetGroup` and `OSGroup`.
+For each ref project and src project, we define `BuildConfigurations`. `BuildConfigurations` is a tuple for the supported `TargetGroup` and `OSGroup`.
 
 
 ie
 
 ref\System.Runtime.csproj
 ```MSBuild
-<DefaultTargetGroup>netstandard1.7</DefaultTargetGroup>
+<DefaultTargetGroup></DefaultTargetGroup>
 <PropertyGroup>
-  <ProjectConfigurations>
-    $(DefaultTargetGroup)|Windows_NT;
-    $(DefaultTargetGroup)|OSX;
-    $(DefaultTargetGroup)|Linux;
-    netcoreapp1.1|Windows_NT;
-    netcoreapp1.1|OSX;
-    netcoreapp1.1|Linux
-  </ProjectConfigurations>
+  <BuildConfigurations>
```

Comment:
Yes please ignore buildingvertical.md I started updating some of that but then decided to cover it in the other doc. DefaultTargetGroup will not necessary.